core/sql/optimizer/BindRelExpr.cpp

```diff
@@ -11046,6 +11046,14 @@ RelExpr *MergeDelete::bindNode(BindWA *bindWA)
         bindWA->setErrStatus();
         return this;
       }
+      if (CmpCommon::getDefault(COMP_BOOL_175) == DF_OFF)
```

Comment:
UPDATE + INSERT does not have the issue; I saw successful TSJ plans with UPDATE + INSERT. With DELETE + INSERT, the TSJ plans fail. DELETE + INSERT was turned off by default before; only by turning on COMP_BOOL_175 to ON would it be allowed. I took this to mean that it was known to be buggy, and left it in this state. I spent about a half day debugging it without success and abandoned that effort for now as it seemed outside the scope of the costing work. I will file a JIRA.